.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: ruby-versions
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+jobs:
+  run:
+    name: ruby
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        version:
+          [
+            "2.2",
```

Comment:
What are the issues running back to 1.9? It seems like it'd be nice to not run both Travis and this if we're going to be adding this. Although perhaps this needs more discussion if we're going to transition the repos over.

Reply:
These are the officially supported versions: https://github.com/ruby/setup-ruby#supported-versions  If we wanted to go back further we could look at installing those earlier versions explicitly or finding Docker images for them for the Ubuntu runs, at least. Since it wasn't in the story, I didn't look into it.

This is my low effort version of covering more operating systems but not sinking in the effort to try to get rid of Travis. In my testing I just deleted the Travis config and ran the GitHub tests. It was much faster and less frustrating.

I think it's probably fine to merge this given that it adds extra coverage for now and we can always delete the workflow config file later if we don't like this or don't feel it's worth maintaining. Travis is just slow a lot of days and GitHub doesn't seem to have the issues with having stuff just linger in the queue.